### ISSUE TITLE
Tiddlers tagged [[OfficialPlugins]] missing source/install info

### ISSUE THREAD

**Issue by 00SS** [Jermolene/TiddlyWiki5#3723]
This issue was brought up in Google Groups tiddlywikidocs: [External Attachments Plugin Correction Suggestion](https://groups.google.com/forum/?hl=en#!topic/tiddlywikidocs/HreSFyBi9-A) on 2019-01-22

13 tiddlers tagged `OfficialPlugins` do not have instructions on what to do next to find that plugin nor how to install it.

To correct that omission, it is proposed to do the following:

A new tiddler would be put here: [Branch:tiddlywiki-com TiddlyWiki5/editions/tw5.com/tiddlers/plugins/](https://github.com/Jermolene/TiddlyWiki5/tree/tiddlywiki-com/editions/tw5.com/tiddlers/plugins)
file: `Where to find plugins.tid`
title: `Where to find plugins`
tag: `Plugins`
text:
```
Plugins can be found by opening the <<.button control-panel>>. On the <<.controlpanel-tab Plugins>> tab, click on the
{{$:/core/ui/ControlPanel/Plugins/AddPlugins}} button, then search for the plugin's name.

For installing a plugin, see [[Installing a plugin from the plugin library]].
```
The 13 tiddlers will be edited to transclude this in at the bottom of each of those tiddlers.

Note: The alternative suggestion was to use a ViewTemplate for tiddlers tagged `OfficialPlugins`

My opinion for this solution is that it would be confusing in later years for those maintaining documentation to keep track of this and perhaps more view templates that would be made. Transclusion is a better solution as usually when a new tiddler is being added, structure reference is taken from already existing tiddlers with the same tags.

**Comment by pmario** [comment]
see:https://tiddlywiki.com/#Installing%20a%20plugin%20from%20the%20plugin%20library

**Comment by 00SS** [comment]
@pmario 
Most of the plugin tiddlers link to an external site that provides a demo. Some don't have any forward links. 

This is a dead-end for how to find and install the plugin for use in one's own TiddlyWiki. It is not intuitive how to do so and requires searching for more information.

It was suggested that a short explanation of how to find and download and install that plugin at the bottom of each of those tiddlers would be useful.

**Comment by bimlas** [comment]
I agree: it is better to have the information in front of the user instead of having to search for it. Fortunately, Tiddly is flexible enough to easily display the tiddler when the conditions are met; why not use this feature?

BTW for some reason, the External Attachments Plugin is only available in prerelease, although it has been there since October 2017 (e646d207c3aa9e7570fedeb27a6576b4d91ea8d9).

**Comment by 00SS** [comment]
Cannot safely use the phrase: **search for the plugin's name** : until #3741 is implemented.

Instead, we can say: **search for the plugin tiddler's title**

Ouch.

**Comment by 00SS** [comment]
@Jermolene 

Since #3741 has been fixed in the 5.1.20 pre-release - I prefer updating the documentation on the **master** branch instead of the **tiddlywiki-com** documentation branch.

This is because the documentation will only be applicable in the new release. Is this acceptable?

**Comment by 00SS** [comment]
***ping*** @Jermolene
